<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="GLcanvas" name="glCanvas" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWidgets">
    <property name="title">
     <string>Widgets</string>
    </property>
    <addaction name="actionWindow"/>
    <addaction name="actionTrimesh"/>
   </widget>
   <addaction name="menuWidgets"/>
  </widget>
  <action name="actionTrimesh">
   <property name="text">
    <string>Trimesh</string>
   </property>
  </action>
  <action name="actionSkel">
   <property name="text">
    <string>Skel</string>
   </property>
  </action>
  <action name="actionWindow">
   <property name="text">
    <string>Window</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLcanvas</class>
   <extends>QWidget</extends>
   <header>GUI/glcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
